pattern-1/artifacts/apim-analytics/wso2apim-analytics-1-deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
     spec:
       hostname: wso2apim-analytics-1
       containers:
-      - image: docker.wso2.com/wso2am-analytics-kubernetes:2.1.0
+      - image: wso2am-analytics-kubernetes:2.2.0
```

Comment:
The wso2 docker registry hostname was used purposely in the dock image id for allowing users to directly pull docker images from the official repository. Might be better to preserve it.